packages/core/EventManager.ts

```diff
@@ -97,7 +99,7 @@ export default class EventManager {
     // (type closecom_other_calendar)
     this.calendarCredentials = appCredentials.filter(
       // Backwards compatibility until CRM manager is implemented
-      (cred) => cred.type.endsWith("_calendar") || cred.type.endsWith("_crm")
+      (cred) => cred.type.endsWith("_calendar") && !cred.type.includes("other_calendar")
```

Comment:
Older crm credentials end in `other_calendar` so let's not lump them in with the calendar credentials